app/controllers/api/v1/yelp/businesses_search_controller.rb

```diff
@@ -3,4 +3,14 @@ def utilities
     data = BusinessesSearchFacade.utilities_search(params[:location], params[:type])
     render json: BusinessesSerializer.new(data)
   end
+
+  def recreation
+    data = BusinessesSearchFacade.recreation_search(params[:location], params[:type])
+    render json: BusinessesSerializer.new(data)
+  end
+
+  def find_business
+    data = BusinessesSearchFacade.find_business(params[:yelp_business_id])
+    render json: BusinessesSerializer.new(data)
+  end
 end
```

Comment:
Im wondering if we can just cache the results from the index and pull that info into the show page instead of making aother API call for info we already have